pom.xml

```diff
@@ -363,6 +366,64 @@
         </repository>
     </repositories>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <id>cornell-internal</id>
+            <url>https://maven.cit.cornell.edu/repository/internal</url>
```

Comment:
What's this repository?  Should we use our group's repo: nexus.cucloud.net

Reply:
The plugin repository entries were added to the POM by copying and tweaking all of its regular repository entries. This was one of those existing repository entries in the POM.

If you believe these entries need further cleanup, we could add a follow-up user story to do so unless you think it's urgent to address them now.

Reply:
OK, I see them know.  I think a follow up user story to investigate these entries make sense.  Since these entries were pre-existing and this branch builds, we don't need to adjust this as apart of the kualico patch

Reply:
I created KFSPTS-25932 to perform more cleanup work on these repository entries.